public/global.css

```diff
@@ -1,62 +1,147 @@
-html, body {
-	position: relative;
-	width: 100%;
-	height: 100%;
+html,
+body {
+  position: relative;
+  width: 100%;
+  height: 100%;
 }
 
 body {
-	color: #333;
-	margin: 0;
-	padding: 8px;
-	box-sizing: border-box;
-	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  --NAD_BLACK: #323643;
+  --NAD_SERVICE: #979797;
+  --NAD_GREY: #7a7a7a;
+  --NAD_WHITE: #ffffff;
+  --NAD_MILK: #e6e6e6;
+  --NAD_BLUE: #0960bd;
+  --NAD_BG: #d8d8d8;
+
+  color: #333;
+  margin: 0;
+  padding: 8px;
+  box-sizing: border-box;
+  font-family: Poppins, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
 }
 
 a {
-	color: rgb(0,100,200);
-	text-decoration: none;
+  color: rgb(0, 100, 200);
+  text-decoration: none;
 }
 
 a:hover {
-	text-decoration: underline;
+  text-decoration: underline;
 }
 
 a:visited {
-	color: rgb(0,80,160);
+  color: rgb(0, 80, 160);
 }
 
 label {
-	display: block;
+  display: block;
 }
 
-input, button, select, textarea {
-	font-family: inherit;
-	font-size: inherit;
-	padding: 0.4em;
-	margin: 0 0 0.5em 0;
-	box-sizing: border-box;
-	border: 1px solid #ccc;
-	border-radius: 2px;
+input,
+button,
+select,
+textarea {
+  font-family: inherit;
+  font-size: inherit;
+  padding: 0.4em;
+  margin: 0 0 0.5em 0;
+  box-sizing: border-box;
+  border: 1px solid #ccc;
+  border-radius: 2px;
 }
 
 input:disabled {
-	color: #ccc;
+  color: #ccc;
 }
 
 input[type="range"] {
-	height: 0;
+  height: 0;
 }
 
 button {
-	color: #333;
-	background-color: #f4f4f4;
-	outline: none;
+  color: #333;
+  background-color: #f4f4f4;
+  outline: none;
 }
 
 button:active {
-	background-color: #ddd;
+  background-color: #ddd;
 }
 
 button:focus {
-	border-color: #666;
+  border-color: #666;
+}
+
+h1,
+h2,
+h3 {
+  color: var(--NAD_BLACK);
+  font-family: Poppins;
+}
+
+h1 {
+  font-family: Poppins Black;
+  font-size: 24px;
+  font-weight: 900;
+}
+
+h2 {
+  font-size: 26px;
+  font-family: Poppins Medium;
+}
+
+input::placeholder {
+  /* font-family: Poppins; */
+}
+.container {
+  margin: 0 auto;
+  padding: 0 16px;
+}
+.play-btn:hover svg path {
+  fill: var(--NAD_WHITE);
+  stroke: var(--NAD_BLACK);
+}
+.profile-btn {
+  background-color: #fff;
+  border: none;
+  box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.05);
+  cursor: pointer;
+  font-family: Poppins;
+  font-size: 16px;
+  height: 63px;
+  line-height: 63px;
+  padding: 0 20px;
+  text-align: left;
+  width: 100%;
+}
+.user {
```

Comment:
why ```.user``` styles here?
can we move it to user component?
and all styles related to user below